resources/lang/en/common.php

```diff
@@ -28,7 +29,14 @@
     'subfleet'       => 'Subfleet',
     'distance'       => 'Distance',
     'fuel'           => 'Fuel',
+    'fuelonboard'    => 'Fuel On Board',
```

Comment:
Add these into the other lang files too with Google Translate or something

Reply:
This is why i hate translations :)

Reply:
Haha me too

Reply:
Done